lib/Model/Attachment.php

```diff
@@ -47,6 +47,8 @@ class Attachment extends Entity {
 	public const TYPE_MEDIA = 'media';
 	public const TYPE_OTHER = 'other';
 	public const TYPE_POLL = 'poll';
+	public const TYPE_RECORD_AUDIO = 'record-audio';
```

Comment:
Needs to be documented in https://github.com/nextcloud/spreed/blob/062ed28540c21d09a19cf7077033e1b0165658c4/docs/constants.md#shared-item-types